tests/ChatTests.hs

```diff
@@ -581,6 +582,16 @@ testUpdateProfile =
             bob <## "use @cat <message> to send messages"
         ]
 
+testUpdateProfileImage :: IO ()
+testUpdateProfileImage =
+  testChat2 aliceProfile bobProfile $
+    \alice bob -> do
+      connectUsers alice bob
+      -- Note we currently don't support removing profile image.
+      alice ##> "/profile_image data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAIAQMAAAD+wSzIAAAABlBMVEX///+/v7+jQ3Y5AAAADklEQVQI12P4AIX8EAgALgAD/aNpbtEAAAAASUVORK5CYII="
+      alice <## "profile image updated"
+      (bob </)
```

Comment:
Maybe we could modify `viewContactUpdated` in `View` with a message for the recipient of XInfo that the contact's profile was updated with a new image. It might not make much sense for terminal to print this message, but then at least it could be printed depending on a `testView` parameter in `responseToView` (it's for conditional output to virtual terminal during tests) - then we could test if the recipient received the same image the sender sent.

Reply:
Evgeny and I discussed this and decided that it isn't necessary as the profile update broadcast is already tested for updating profile names and it is the same function so we don't need to retest it here in a case where it isn't used other than for testing. The full story is that I started implementing it and then rolled it back following a the aforementioned discussion. It would be nice but I don't think it's worth holding up this PR for.